### ISSUE TITLE
Assertions for Add/Attach/Detach Relations

### ISSUE THREAD

**Issue by calvinps** [dillingham/nova-assertions#6]
Is there a way to test if you can add/attach/detach relationships? Nova provides a way to authorize editing relationships on a resource: https://nova.laravel.com/docs/2.0/resources/authorization.html#relationships

If not, maybe this would make a good addition to the Policies assertions (e.g. assertCanAdd, assertCanAttach, assertCanDetach)?

**Comment by stevelacey** [comment]
Yeah this would be great, assertion of the pivot fields on add and edit attachment would also be handy

**Comment by dillingham** [comment]
PRs welcome
